incubator/group/types.proto

```diff
@@ -314,5 +314,13 @@ message Params {
 message GenesisState {
     option (gogoproto.equal) = true;
     option (gogoproto.goproto_stringer) = false;
-    Params Params = 1 [(gogoproto.nullable)=false];
+    Params Params = 1 [(gogoproto.nullable) = false];
+    uint64 GroupSeq = 2;
+    bytes Groups = 3 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    bytes GroupMembers = 4 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    uint64 GroupAccountSeq = 5;
+    bytes GroupAccounts = 6 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    uint64 ProposalSeq = 7;
+    bytes Proposals = 8 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    bytes Votes = 9 [(gogoproto.casttype) = "encoding/json.RawMessage"];
 }
```

Comment:
So what if we were to take an approach like this where [AnyGroupAccountMetadata](https://github.com/regen-network/cosmos-modules/blob/e6dbd7981c4e2337f4d695e0c7169421e86e07cb/incubator/group/types.proto#L332) and [AnyProposal](https://github.com/regen-network/cosmos-modules/blob/e6dbd7981c4e2337f4d695e0c7169421e86e07cb/incubator/group/types.proto#L361) come from #63 and leverage `google.protobuf.Any`?

```proto
message GenesisState {
     Params params = 1;
     uint64 group_seq = 2;
     repeated GroupMetadata groups = 3;
     repeated GroupMember group_members = 4;
     uint64 group_account_seq = 5;
     AnyGroupAccountMetadata group_acconts = 6;
     uint64 proposal_seq = 7;
     repeated AnyProposal proposals = 8;
     repeated Vote votes = 9;
 }
```

Reply:
With using concrete types rather than json bytes we can not simply let the orm layer do the en-/decoding. I am wondering now what are the benefits of concrete types here? The genesis is only used for data import/ export.
We won't get additional type safety as this is already covered within orm. But I guess it is easier to read and understand what would be inside the json bytes. Although with `Any` types this is not obvious again.

Reply:
I wonder if you have any more thoughts after seeing the Any proposal? Also are you familiar with how the current SDK genesis encodes interfaces?

Reply:
I may be a bit biased by the implementation already. The orm store requires concrete types and won't work with interfaces. For the genesis in/export I was just focused on "it should work automated by the orm framework layer".  I can spike a protobuf version